### PR TITLE
Feat/deployment

### PR DESCRIPTION
**Pull request by ottonotto**
I redid/re-started from the last approved PR, we need to drop/close the last unapproved PR and not approve the changes, as to allow this PR to be looked at and approved.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const express = require('express');
+const bodyParser = require('body-parser');
+// const { Places } = require('./api/places');
```
Comment (shams-ali) [inline server/app.js:4]: this is not being used...it should not be in this PR

File: server/api/places.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const { Router } = require('express');
```
Comment (shams-ali) [inline server/api/places.js:1]: this file has nothing to do with deployment...do not include in this PR

File: server/config.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const { GITHUB_TOKEN } = process.env;
```
Comment (shams-ali) [inline server/config.js:1]: this file is not being used anywhere....please remove it from the PR

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const dotenv = require('dotenv');
+
+dotenv.config({
+    path: path.resolve(__dirname, '../.env'),
+});
```
Comment (shams-ali) [inline server/index.js:6]: it automatically resolves to .env of root directory...please see npm doc, no need to specify path

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+const { app } = require('./app');
+
+const PORT = 3000;
```
Comment (shams-ali) [inline server/index.js:10]: this should be an ENV VARIABLE

File: server/app.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+const CLIENT_PATH = path.resolve(__dirname, '../build');
+const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(CLIENT_PATH));
+
+// app.use('/api/places', Places);
```
Comment (shams-ali) [inline server/app.js:12]: please clean up your code before making a PR to organization branch

File: .husky/.gitignore [hunk 7]
```diff
@@ -0,0 +1 @@
+_
```
Comment (shams-ali) [inline .husky/.gitignore:1]: what is this file

File: package.json [hunk 8]
```diff
@@ -6,6 +6,10 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "body-parser": "^1.19.1",
```
Comment (shams-ali) [inline package.json:9]: this is built into express...please use express way of doing this

Comment (shams-ali) [inline package.json:9]: please remove this as its not needed for this ticket

File: server/app.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
+const path = require('path');
+const express = require('express');
+const bodyParser = require('body-parser');
+// const { Places } = require('./api/places');
+
+const CLIENT_PATH = path.resolve(__dirname, '../build');
+const app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
```
Comment (shams-ali) [inline server/app.js:9]: you dont need these, its not part of this ticket

File: server/app.js (hunk 1 above)
Comment (ottonotto) [reply server/app.js]: dropped

File: server/api/places.js (hunk 2 above)
Comment (ottonotto) [reply server/api/places.js]: dropped

File: server/config.js (hunk 3 above)
Comment (ottonotto) [reply server/config.js]: dropped

File: server/index.js (hunk 4 above)
Comment (ottonotto) [reply server/index.js]: dropped

File: server/index.js (hunk 5 above)
Comment (ottonotto) [reply server/index.js]: dropped

File: server/app.js (hunk 6 above)
Comment (ottonotto) [reply server/app.js]: dropped

File: .husky/.gitignore (hunk 7 above)
Comment (ottonotto) [reply .husky/.gitignore]: dropped

File: package.json (hunk 8 above)
Comment (ottonotto) [reply package.json]: dropped

Comment (ottonotto) [reply package.json]: dropped

File: server/app.js (hunk 9 above)
Comment (ottonotto) [reply server/app.js]: dropped

**Pull request closed without merging**